x-pack/plugins/security_solution/public/timelines/components/fields_browser/helpers.tsx

```diff
@@ -144,7 +143,7 @@ export const mergeBrowserFieldsWithDefaultCategory = (
   }),
 });
 
-export const getAlertColumnHeader = (timelineId: string, fieldId: string) =>
-  timelineId === TimelineId.detectionsPage || timelineId === TimelineId.detectionsRulesDetailsPage
-    ? alertsHeaders.find((c) => c.id === fieldId) ?? {}
-    : {};
+export const getAlertColumnHeader = (fieldId: string) =>
```

Comment:
I think we should keep `timelineId`, because this change breaks Fields browser on DE page
![image](https://user-images.githubusercontent.com/5188868/101544846-a6c1b880-39a6-11eb-977f-b6f862b11e30.png)

Reply:
ok, rolled back. so in this PR we can still see the rule name in the events table on timelines' page, but not able to click and link to the rule's page.

Reply:
<img width="1047" alt="Screenshot 2020-12-09 at 22 23 57" src="https://user-images.githubusercontent.com/6295984/101641860-3197c700-3a2a-11eb-8298-5dacaa16c273.png">